.github/workflows/test.yml

```diff
@@ -11,7 +11,10 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
-      - uses: azure/setup-helm@v1
+      - uses: azure/setup-helm@v3
+        with:
+          version: '3.9.0'
+        id: install
```

Comment:
Actually, GitHub has nowadays Helm v3 preinstalled (currently 3.10.1). So we can completely remove this action
```suggestion
```